jest.setup.js

```diff
@@ -13,10 +13,30 @@ expect.addSnapshotSerializer(
   })
 );
 
+const toHaveBooleanAttribute = attribute => wrapper => {
+  // :hidden="true" becomes
+  // hidden="hidden" in Vue 2 and
+  // hidden="" in Vue 3.
+
+  // So we need this to write correct tests to match them in both versions.
+  const value = wrapper.attributes(attribute);
+  if ((isVue2 && value === attribute) || (isVue3 && value === '')) {
+    return { pass: true };
+  } else {
+    return {
+      pass: false,
+      message: () => `expected ${wrapper} to have \`${attribute}\` attribute`,
+    };
+  }
+};
+
 expect.extend({
-  toHaveEmptyHTML: received => {
-    const html = received.html();
-    if ((isVue2 && html === '') || (isVue3 && html === '<!---->')) {
+  toHaveEmptyHTML: wrapper => {
```

Comment:
they should all be declared outside I think, for consistency

Reply:
https://github.com/algolia/vue-instantsearch/pull/1021/commits/a43c889402532f9bfda816fed1ffc45a9225781d